frontend/components/feedback-task/header/metadata-filter/MetadataFilter.vue

```diff
@@ -93,11 +99,38 @@ export default {
         this.metadataFilters.convertToRouteParam()
       );
     },
+    openCategoryFilter(category, e) {
+      e.stopPropagation();
+      this.visibleDropdown = this.visibleDropdown ? false : true;
+      this.selectMetadataCategory(category);
+    },
+    removeCategoryFilters(category, e) {
+      e.stopPropagation();
+      const clickedCategory = this.metadataFilters.metadata.find(
+        (f) => f.name === category
+      );
+      if (clickedCategory.isTerms) {
+        this.metadataFilters.metadata
+          .find((f) => f.name === category)
+          .options.map((opt) => (opt.selected = false));
+      }
+
+      this.applyFilter();
+    },
   },
   computed: {
     metadataCategoriesName() {
       return this.metadataFilters.categories;
     },
+    metadataFilterFromQuery() {
+      return this.$route.query?._metadata?.split("+") ?? [];
```

Comment:
please review this line, because is duplicated in the parent component, I don't know if you had time to read my two latest commit, because I created a useRoute service to manage this case